lua/neogit/lib/git/stash.lua

```diff
@@ -65,54 +59,67 @@ local function perform_stash(include)
   end
 end
 
-local function update_stashes(state)
-  local result = cli.stash.args("list").call():trim()
-  state.stashes.items = parse(result.stdout)
+function M.list_refs()
+  local result = cli.reflog.show.format("%h").args("stash").call_ignoring_exit_code():trim()
+  if result.code > 0 then
+    return {}
+  else
+    return result.stdout
+  end
+end
+
+function M.stash_all(args)
+  cli.stash.arg_list(args).call()
+  -- this should work, but for some reason doesn't.
+  --return perform_stash({ worktree = true, index = true })
 end
 
-local function list()
-  return cli.reflog.show.format("%h").args("stash").call():trim().stdout
+function M.stash_index()
```

Comment:
I also prefer the `M.` syntax rather than the exports table at the end.

Makes *goto definition* jump to it immediately rather than landing on the field of the exports table.

Good work

Reply:
Exactly!